img/h/huobi.com.svg

```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 18.8 29.7"><path d="M9.1,0H8.9c0,4.2-1.1,6.8-2.5,8.7s-2.9,3.3-4.1,4.8a9.7,9.7,0,0,0-1.6,10A9.9,9.9,0,0,0,7,29.5c1.4.4,1.2.1.6-1a13,13,0,0,1-1.7-5c-.3-2.7,1.4-5.2,3.2-7.5s3.6-4.6,3.6-6.9h0a14.5,14.5,0,0,0-1.2-5.7A9.9,9.9,0,0,0,9.1,0Z" fill="#d7e2f2"/><path d="M15.2,12h-.1a18,18,0,0,1-2.5,5.3c-4.6,6.9-2,10.2-.5,12s0,0,2.1-1.1a9.6,9.6,0,0,0,4.6-7.3A10.9,10.9,0,0,0,15.2,12" fill="#357ce1"/></svg>
+<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 132.5 202.6"><path fill="#00003e" d="M87 62.1c.7-29.8-16.2-55.7-25-62 0 0-.8-.5-.7.7-1 56.4-30 71.5-45.6 92.2a67.8 67.8 0 0 0 32 108.1c.6.2 2.3.7 5.6 1.4 1.8.4 2.3-1.1 1-3.3-4.5-7.8-12.5-21-14.1-38.1-3.6-39.5 46.2-64.3 46.9-99z"/><path fill="#008cd6" d="M106.1 81.2c-.3-.2-.7-.2-.8.2-.8 7.2-8.2 22-18 36-32.8 46.8-16.2 68-3.4 82.7 2.4 2.8 3.6 2.2 4.8.2 1.2-1.8 3-4.2 10.5-7.8a65.5 65.5 0 0 0 33-50.4 71.5 71.5 0 0 0-26-60.9z"/></svg>
```

Comment:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 133 203"><path fill="#fff" d="M61 1c-1 56-30 71-45 92a68 68 0 0 0 32 108l5 1c2 1 3-1 1-3-4-8-12-21-14-38-3-39 46-64 47-99C88 32 71 6 62 0l-1 1Z"/><path fill="#008cd6" d="M106 81h-1c-1 8-8 22-18 36-32 47-16 68-3 83 2 3 3 2 5 0 1-1 3-4 10-7a66 66 0 0 0 33-51 72 72 0 0 0-26-61Z"/></svg>
```

Reply:
Now the left side of the logo is white, but it actually should be dark blue. On their website it's only white due to the black background.

Reply:
Change `#fff` to `#00003e` then?